src/main/java/no/skatteetaten/aurora/filter/logging/AuroraHeaderFilter.java

```diff
@@ -29,12 +29,14 @@ public void init(FilterConfig filterConfig) {
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
         throws IOException, ServletException {
         try {
-            copyHeadersFromRequestToMdc((HttpServletRequest) request, HEADERS);
-            assertKorrelasjonsIdIsSet();
+            try {
```

Comment:
It is very unlikely that the two methods in this try block will throw an Exception. I have never seen it. So just remove the entire catch block. try/finally witht the two methods in there.